tsfile/src/main/java/org/apache/iotdb/tsfile/write/TsFileWriter.java

```diff
@@ -72,6 +73,7 @@
    **/
   private long recordCountForNextMemCheck = 100;
   private long chunkGroupSizeThreshold;
+  private long version = 0;
```

Comment:
I think you can add some comments about the version.
Although there is no comments for ```recordCountForNextMemCheck``` and ```chunkGroupSizeThreshold```.

Reply:
Added, though it is hard to explain a few words.